Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```

Comment:
please use gem "mysql2", delete sqlite3

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/87749736/130193527-69d1ae36-b2aa-43e4-9b35-117bb094c0dd.png)
![Screenshot from 2021-08-18 08-51-10](https://user-images.githubusercontent.com/87749736/130193641-c6e8d278-4aa3-4aab-9a71-594d88abfae7.png)
but i cannot open my rails after change the gem

Reply:
![image](https://user-images.githubusercontent.com/87749736/130205408-9afd376f-b41d-473e-9f3d-57aac85f08ee.png)
done